apps/api/src/app/events/usecases/parse-event-request/parse-event-request.usecase.ts

```diff
@@ -56,7 +55,7 @@ export class ParseEventRequest {
       throw new UnprocessableEntityException('workflow_not_found');
     }
 
-    if (!template.active || template.draft) {
```

Comment:
after change users may still have those "broken" templates with active=true and draft=true.

UI after my changes will show that template is active, but API will be unable to trigger event becasue `draft` is true.